### PR TITLE
[FIX] order is null

### PR DESCRIPTION
**Pull request by GabbasovDinar**

### CODE REVIEW

**Comment by yelizariev** [comment]
Fix lints please

```
pos_fiscal_floor/__openerp__.py:2: [C8103(manifest-deprecated-key), ] Deprecated key "description" in manifest file
pos_invoice_pay/__init__.py:1: [W7993(js-empty-coverage), ] JS files are not covered (phantom_js is not used). Please add js tests: https://odoo-development.readthedocs.io/en/latest/qa/js/index.html
pos_invoice_pay/data.xml:5: [W7939(deprecated-data-xml-node), ]  Use <odoo> instead of <odoo><data> or use <odoo noupdate="1">instead of <odoo><data noupdate="1">
pos_invoice_pay/views.xml:5: [W7939(deprecated-data-xml-node), ]  Use <odoo> instead of <odoo><data> or use <odoo noupdate="1">instead of <odoo><data noupdate="1">
pos_invoice_pay/actions/base_action_rules.xml:5: [W7939(deprecated-data-xml-node), ]  Use <odoo> instead of <odoo><data> or use <odoo noupdate="1">instead of <odoo><data noupdate="1">
```

File: pos_invoice_pay/README.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Credits
 
 Contributors
 ------------
-* `Artyom Losev <https://it-projects.info/team/ArtyomLosev>`__
+* `Artyom Losev <https://it-projects.info>`__
```
Comment (yelizariev) [inline pos_invoice_pay/README.rst]: Keep original link please, even though it's 404 for now

**Review by yelizariev — CHANGES_REQUESTED** [review]

File: pos_invoice_pay/README.rst (hunk 1 above)
Comment (ilmir-k) [reply pos_invoice_pay/README.rst]: @yelizariev done

**Comment by yelizariev** [comment]
It has new error now

`<p class="title">TypeError: undefined is not a function (evaluating 'line_ids.includes(lines[i].id)')</p>`

**Comment by yelizariev** [comment]
Related: https://github.com/it-projects-llc/pos-addons/pull/687

**Comment by GabbasovDinar** [comment]
@yelizariev Can you check the travis? I don't understand why I have the errors (in tests)

**Comment by yelizariev** [comment]
It has error with  ``pos_orderline_absolute_discount``
```
2018-05-04 11:53:27,686 5276 INFO openerp_test odoo.tests.common: phantomjs: Tour pos_orderline_absolute_discount_tour: step .button.pay succeeded
2018-05-04 11:53:27,692 5276 INFO openerp_test odoo.tests.common: phantomjs: MS Main response #8: [object Object]
2018-05-04 11:53:27,712 5276 INFO openerp_test odoo.tests.common: phantomjs: Tour pos_orderline_absolute_discount_tour: step .modal-dialog.cashier .selection-item succeeded
2018-05-04 11:53:27,732 5276 INFO openerp_test odoo.tests.common: phantomjs: Tour pos_orderline_absolute_discount_tour: step .paymentmethod:contains("Cash") succeeded
2018-05-04 11:53:27,752 5276 INFO openerp_test odoo.tests.common: phantomjs: Tour pos_orderline_absolute_discount_tour: step .payment-numpad .input-button:contains("2"):visible succeeded
2018-05-04 11:53:27,770 5276 INFO openerp_test odoo.tests.common: phantomjs: Tour pos_orderline_absolute_discount_tour: step .button.next:visible succeeded
2018-05-04 11:53:37,783 5276 INFO openerp_test odoo.tests.common: phantomjs: _check_for_tooltip
2018-05-04 11:53:37,783 5276 INFO openerp_test odoo.tests.common: phantomjs: - modal_displayed: 0
2018-05-04 11:53:37,784 5276 INFO openerp_test odoo.tests.common: phantomjs: - trigger '.js_connecting:visible': 0
2018-05-04 11:53:37,785 5276 INFO openerp_test odoo.tests.common: phantomjs: - visible trigger '.js_connecting:visible': 0
2018-05-04 11:53:37,786 5276 INFO openerp_test odoo.tests.common: phantomjs: _check_for_tooltip
2018-05-04 11:53:37,787 5276 INFO openerp_test odoo.tests.common: phantomjs: - modal_displayed: 0
2018-05-04 11:53:37,787 5276 INFO openerp_test odoo.tests.common: phantomjs: - trigger '.js_connecting:visible': 0
2018-05-04 11:53:37,788 5276 INFO openerp_test odoo.tests.common: phantomjs: - visible trigger '.js_connecting:visible': 0
2018-05-04 11:53:40,144 5276 INFO openerp_test odoo.tests.common: phantomjs: <body class="">
<div class="o_main_content"><div class="o_control_panel o_hidden"></div><div class="o_content"><div class="pos">
```

**Comment by yelizariev** [comment]
`2018-05-04 11:43:18,077 32757 WARNING 16275-677-6c564b-base odoo.addons.base.module.module: module pos_fiscal_floor: description is empty !
`

**Comment by yelizariev** [comment]
```
************ Module pos_fiscal_floor
pos_fiscal_floor/__init__.py:1: [E7995(absent-icon), ] File: static/description/icon.png is absent in module. Get possible icons here: https://odoo-development.readthedocs.io/en/latest/dev/docs/icon.png.html
pos_fiscal_floor/__init__.py:1: [E7997(absent-changelog), ] File: doc/changelog.rst is absent in module. Get a template here: https://odoo-development.readthedocs.io/en/latest/dev/docs/changelog.rst.html
pos_fiscal_floor/__init__.py:1: [E7998(absent-doc), ] File: doc/index.rst is absent in module. Get a template here: https://odoo-development.readthedocs.io/en/latest/dev/docs/usage-instructions.html
pos_fiscal_floor/__init__.py:1: [W7993(js-empty-coverage), ] JS files are not covered (phantom_js is not used). Please add js tests: https://odoo-development.readthedocs.io/en/latest/qa/js/index.html
pos_fiscal_floor/__init__.py:1: [C7902(missing-readme), ] Missing ./README.rst file. Template here: https://odoo-development.readthedocs.io/en/latest/dev/docs/__openerp__.py.html

```

**Comment by GabbasovDinar** [comment]
fixed here: https://github.com/it-projects-llc/pos-addons/pull/693

**Pull request closed without merging**